src/Playwright.Tests.TestServer/SimpleServer.cs

```diff
@@ -45,12 +48,14 @@ public class SimpleServer
     {
         const int MaxMessageSize = 256 * 1024;
 
-        private readonly IDictionary<string, Action<HttpContext>> _subscribers;
```

Comment:
this was only used in one place, got rid of it and used SetRoute like we do upstream.